composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.5",
-        "elasticsearch/elasticsearch": "^2.3"
+        "elasticsearch/elasticsearch": "5.0.x-dev"
```

Comment:
Branch 5.0 is now stable
https://github.com/elastic/elasticsearch-php#installation-via-composer